warp10/src/test/java/io/warp10/script/WarpScriptLibHasSameNamesTest.java

```diff
@@ -0,0 +1,1644 @@
+//
```

Comment:
What is the use of this test class beyond checking once that your PR did not introduce naming errors?

Reply:
None, I will remove it when PR is ready to be accepted